utils/sparse.sh

```diff
@@ -35,12 +35,27 @@ RE="$RE|error: undefined identifier '__builtin_fpclassify'"
 search=$(gcc -print-search-dirs | awk '($1 == "install:"){print "-I" $2}')
 
 #
-# We're trying to use sparse against glibc headers which go wild trying to
```

Comment:
Don't remove the comment that describes  why this nonsense is in here.  The text you ended up with should replace the "But not __SIZE_TYPE" as that's all that you're changing, and leave the initial text.